### PR TITLE
[Test automation] re-enable the skipped e2e tests

### PR DESCRIPTION
**Pull request by gustavolira**
## Description

Re-enable the e2e tests that are temporarily marked as 'skip'.

## Which issue(s) does this PR fix

- Fixes Tests which depends on OCM plugin and cluster token authentication 

## PR acceptance criteria

All tests passing

### CODE REVIEW

File: .ibm/pipelines/auth/secrets-rhdh-secrets.yaml [hunk 1]
```diff
@@ -14,7 +14,8 @@ data:
   GITHUB_ORG: amFudXMtcWU=
   K8S_CLUSTER_NAME: bXktY2x1c3Rlcgo=
   K8S_CLUSTER_API_SERVER_URL: aHR0cHM6Ly9jMTAwLWUudXMtZWFzdC5jb250YWluZXJzLmNsb3VkLmlibS5jb206MzIyMTIK
-  K8S_SERVICE_ACCOUNT_TOKEN: dGVtcA==
+  K8S_CLUSTER_TOKEN: dGVtcA==
```
Comment (josephca) [inline .ibm/pipelines/auth/secrets-rhdh-secrets.yaml:17]: @gustavolira do we need to have `K8S_CLUSTER_TOKEN` in this file? if not, can we remove it? I noticed that `K8S_CLUSTER_TOKEN` is used with 'oc' command in `.ibm/pipelines/openshift-ci-tests.sh` but the real value is set by `.ibm/pipelines/env_variables.sh`

**Review by josephca — CHANGES_REQUESTED** [review]

File: .ibm/pipelines/resources/cluster_role_binding/cluster-role-binding-ocm.yaml [hunk 2]
```diff
@@ -7,6 +7,9 @@ roleRef:
   kind: ClusterRole
   name: rhdh-k8s-plugin-ocm
 subjects:
-- kind: ServiceAccount
-  name: rhdh-k8s-plugin
-  namespace: showcase
+  - kind: ServiceAccount
+    name: rhdh-k8s-plugin
+    namespace: showcase
```
Comment (subhashkhileri) [inline .ibm/pipelines/resources/cluster_role_binding/cluster-role-binding-ocm.yaml:12]: Will this also work with the nightly namespace/job? The namespace will change to showcase-ci-nightly.

Comment (gustavolira) [reply .ibm/pipelines/resources/cluster_role_binding/cluster-role-binding-ocm.yaml:12]: @subhashkhileri updated, thanks for the heads up

**Review by josephca — APPROVED** [review]
/lgtm
/approve

**Pull request merged**